applications/Unity.GrantManager/src/Unity.GrantManager.Application/Integrations/OrgBook/OrgBookService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Newtonsoft.Json;
+using RestSharp;
+using System.Threading.Tasks;
+using Unity.GrantManager.Integration.Orgbook;
+using Unity.GrantManager.Integrations.Exceptions;
+using Unity.GrantManager.Integrations.Http;
+using Volo.Abp;
+
+namespace Unity.GrantManager.Integrations.Orgbook
+{
+    [IntegrationService]
+    public class OrgBookService : GrantManagerAppService, IOrgBookService
+    {
+        private readonly IResilientHttpRequest _resilientRestClient;
+
+        private readonly string orgbook_base_api = "https://orgbook.gov.bc.ca/api/v4";
```

Comment:
this should be moved to appsettings.json

Reply:
Daryl does not want them in App settings, I guess I can add this to the configuration but it is not going to change.

Reply:
we can leave it for now, but definitely tech debt and should be looked at with the next tech debt sprint